packages/ui-components/src/components/radio-list/radio-list.tsx

```diff
@@ -0,0 +1,50 @@
+import { Component, Event, EventEmitter, Host, Prop, h } from '@stencil/core';
+import { IRadioList, IRadioListEvents } from './radio-list.types';
+import { IRadioListItem } from '../radio-list-item/radio-list-item.types';
+
+@Component({
+	tag: 'kv-radio-list',
+	styleUrl: 'radio-list.scss',
+	shadow: true
+})
+export class KvRadioList implements IRadioList, IRadioListEvents {
+	/** @inheritdoc */
+	@Prop({ reflect: true }) label?: string;
+	/** @inheritdoc */
+	@Prop({ reflect: true }) options!: IRadioListItem[];
+	/** @inheritdoc */
+	@Prop({ reflect: true }) selectedOption?: string | number;
+	/** @inheritdoc */
+	@Prop({ reflect: true }) disabledOptions?: Record<string | number, boolean>;
+	/** @inheritdoc */
+	@Prop({ reflect: true }) required?: boolean = false;
+
+	/** @inheritdoc */
+	@Event() optionSelected: EventEmitter<string | number>;
+
+	private onOptionClick = ({ detail }: CustomEvent<string | number>) => {
+		this.optionSelected.emit(detail);
+	};
+
+	render() {
+		return (
+			<Host>
+				<div class="radio-list-container">
+					{this.label && <kv-form-label label={this.label} required={this.required} />}
+					{this.options.map(item => {
+						return (
+							<kv-radio-list-item
+								{...item}
```

Comment:
what item has more to destruct?

Reply:
`label` and `description`, just these 2 props since it's just 2 props, I'll make them explicit here